README.md

```diff
@@ -80,7 +80,7 @@ If you want the package detection to work, make sure you attach packages to the
  * macOS package name should start with "osx-"
  * linux package name should start with "lin-"
 
-For an real life example, check [a release](https://github.com/Gibberlings3/Tweaks-Anthology/releases/latest) of Tweaks Anthology.
+For an real life example, check [latest release](https://github.com/Gibberlings3/Tweaks-Anthology/releases/latest) of Tweaks Anthology.
```

Comment:
please revert this, but there's also the stray "an".

Reply:
done

Reply:
revert = undo. The github release stream has all releases ever made, so just looking at the last may not show everything. I'll add a suggestion to the change.

Reply:
done